docs/Using-Pantheon/RPC-PubSub.md

```diff
@@ -19,7 +19,7 @@ The RPC Pub/Sub methods are:
     
 ### Using RPC Pub/Sub
 
-The RPC Pub/Sub API requires a persistent connection. Connect to the RPC Pub/Sub API using a tool such as [WebSockets](../Reference/Using-JSON-RPC-API.md#endpoint-address-and-port). 
+The RPC Pub/Sub API requires a persistent connection. Connect to the RPC Pub/Sub API using a tool such as [WebSockets](../JSON-RPC-API/Using-JSON-RPC-API.md#endpoint-address-and-port). 
```

Comment:
I know that this is not part of this PR but I'll say it anyway :)

This sentence gives me the impression that we support another way than just WebSockets to use the Pub-Sub API. In reality, WebSockets is the only thing that we support.

Reply:
Updated